tools/tune/drc/example_drc.m

```diff
@@ -31,6 +31,14 @@ function example_drc()
 params.enabled = 1;
 drc_coefs_and_config_export(params, 'enabled');
 
+% Export experimental configuration for a small speaker
```

Comment:
In my view the only purpose for these lines is to generate the config "threshold_-25_knee_15_ratio_10_post_3", they are not supposed to land upstream. Alternatively, the applied param set could be put into the comment together with the generated config files.
Another point is that the generated config will be changed by any parameter which is not specified here, e.g. attack and release.

Reply:
Yep, there's more parameters. I guess I should change the file name, but the intention of this is to become the default notebook speaker (hda-generic PC) DRC so used by their default topology so it should be in upstream. The parameters are still likely to change with more subjective testing. I used these parameters for an internal demo to show how DRC can be used to make a very audible loudness change. The EQ in the pipeline is also part of the enhance effect.

Reply:
Thanks. It should be a good idea to separate them into multiple files, e.g. example_drc_default.m, example_drc_notebook.m, and others for different use cases. I found that the same way is applied under eq/ and tfdb/

Reply:
I agree, that would be a small and ease maintenance of the tools files.